Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM python:3.7
+
+#declare envs
+ARG SECRET_KEY
+ARG DB_NAME
+ARG DB_USER
+ARG DB_PASSWORD
+ARG DB_HOST
+ARG DB_PORT
+
+#load envs
+ENV SECRET_KEY=${SECRET_KEY} \
+    DB_NAME=${DB_NAME} \
+    DB_USER=${DB_USER} \
+    DB_PASSWORD=${DB_PASSWORD} \
+    DB_HOST=${DB_HOST} \
+    DB_PORT=${DB_PORT}
+
+RUN mkdir /postit
```

Comment:
No need for this line as the WORKDIR command creates the directory if it does not exist https://docs.docker.com/engine/reference/builder/#workdir